### ISSUE TITLE
installation documentation appears to be out of date

### ISSUE THREAD

**Issue by d4l3k** [volcano-sh/website#109]
I was trying to follow the guide at https://volcano.sh/en/docs/installation/ but it doesn't appear to work for the recent releases. 

The docs mention volcano-{Version}-linux-gnu.tar.gz and volcano-{Version}.yaml but the recent releases don't have a release version just the source code + volcano-development.yaml.

The "quick start" from the readme worked

```
kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
```

https://github.com/volcano-sh/volcano#install-with-yaml-files

**Comment by shinytang6** [comment]
The information on the volcano website is a bit outdated indeed.
/cc @Thor-wl

**Comment by Thor-wl** [comment]
> The information on the volcano website is a bit outdated indeed.
> /cc @Thor-wl

Yes, @renhuanyu is working for the update now.  Thanks for the report.
